docs/getting-started/auth-self-defence-checks.adoc

```diff
@@ -48,6 +48,14 @@ To detect if the app is being run on an emulator the `SecurityCheckType#IS_EMULA
 
 To detect if a device has a lock screen set (with pin, fingerprint, pattern) the `SecurityCheckType#HAS_SCREENLOCK` function can be used.  This function uses Android's link:https://developer.android.com/reference/android/app/KeyguardManager.html[KeyguardManager] class.
 
+==== link:javadoc[ALLOW_BACKUP_ENABLED]
```

Comment:
Are all these links meant to be broken?

Reply:
Good idea, I  will remove the javadoc links now :+1: